crates/holochain/src/conductor/api/error.rs

```diff
@@ -49,10 +48,6 @@ pub enum ConductorApiError {
     #[error(transparent)]
     DatabaseError(#[from] DatabaseError),
 
-    /// Workspace error.
-    #[error(transparent)]
-    WorkspaceError(#[from] WorkspaceError),
```

Comment:
This isn't used for anything useful. It was the start of some idea about re-working the "workflow workspace" idea into something more explicit but was just dangling.